appveyor.yml

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I presume that we'll also need some more stuff to get Bazel installed?

Can we split this PR into two: one that fixes the issue, and one that sets up Appveyor? Then we can test that it actually succeeds on Appveyer and merge that in